### PR TITLE
Add Kanagawa

### PR DESCRIPTION
**Pull request by NarutoXY**

### CODE REVIEW

File: schemes/default.nix [hunk 1]
```diff
@@ -143,6 +143,7 @@
   icy = import ./icy/icy.nix;
 
   katy = import ./katy/katy.nix;
+  kanagawa = import ./kanagawa/kanagawa.nix;
```
Comment (Misterio77) [inline schemes/default.nix:146]:
```suggestion

  kanagawa = import ./kanagawa/kanagawa.nix;
```
This should not be grouped together with katy

File: schemes/kanagawa/kanagawa.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  slug = "Kanagawa";
```
Comment (Misterio77) [inline schemes/kanagawa/kanagawa.nix:2]:
```suggestion
  slug = "kanagawa";
```
And the slug must be all lowercase

File: schemes/kanagawa/kanagawa.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  name = "Kanagawa";
+  author = "rebelot";
```
Comment (Misterio77) [inline schemes/kanagawa/kanagawa.nix:4]:
```suggestion
  author = "Tommaso Laurenzi (https://github.com/rebelot)";
```
It's better if this is more detailed, like this

**Review by Misterio77 — CHANGES_REQUESTED** [review]
Thank you! Just a few nits before we merge

**Comment by NarutoXY** [comment]
Committed all the suggestions :D

**Review by Misterio77 — APPROVED** [review]
Looks great! Thank you :))

**Pull request merged**